wasmapi/go/log.go

```diff
@@ -40,11 +47,15 @@ func log(level logLevel, message string) {
 }
 
 func Log(level logLevel, args ...any) {
-	log(level, fmt.Sprint(args...))
+	if gadgetGetLogLevel() >= uint32(level) {
```

Comment:
Do you think it's a good idea to cache the logLevel to void calling this function each time?